packages/genesys-spark/src/index.ts

```diff
@@ -34,9 +38,11 @@ export function loadSparkFonts(): Promise<void> {
  * application bootstrap if it rejects. Its primary use should be for logging
  * unexpected failures.
  */
-export function registerSparkComponents(): Promise<void> {
+export function registerSparkComponents(opts: registerOptions): Promise<void> {
   const SCRIPT_PATH = 'genesys-webcomponents.esm.js';
-  const STYLE_PATH = 'genesys-webcomponents.css';
+  const STYLE_PATH = opts.theme
```

Comment:
@321gillian this is what is causing the error in the console you spotted. I will fix it.